### PR TITLE
Removed solutions tab, horizon_extensions subtree

### PR DESCRIPTION
**Pull request by cloudnull**
This commit removes the horizon-extensions subtree,
as well as the bits in the horizon_extenstions playbook
related to deploying the solutions tab.

Closes-Bug: #1996
(cherry picked from commit 88859c2a9e005cf53a331ce5de7c0be2582a76ba)
(cherry picked from commit a26b3ec3c3f73b5517778f378e96ab3fed8a1438)
Signed-off-by: Kevin Carter <kevin.carter@rackspace.com>

### CODE REVIEW

**Comment by cloudnull** [comment]
recheck_cit_all

**Comment by BjoernT** [comment]
Thanks but we would have to tag a release to make this useful, so we could consume it. But it is unlikely that we install/update another liberty environment ? But a tag might be useful nonetheless to include the MAAS changes?

**Comment by alextricity25** [comment]
@BjoernT I've been battling with this question myself for the past month or so. I'll coordinate with the tiger team that worked on the MaaS changes to see when it would be best to create the next patch release.

**Comment by cloudnull** [comment]
@BjoernT You could checkout a given SHA, assuming we're not able to create a new tag.

**Comment by cloudnull** [comment]
recheck_cit_all

File: scripts/deploy-rpc-playbooks.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```
Comment (major) [inline scripts/deploy-rpc-playbooks.sh]: Should this file be in the cherry pick?

**Comment by odyssey4me** [comment]
It looks like this may need a rebase.

**Comment by cloudnull** [comment]
recheck_cit_all

**Comment by hughsaunders** [comment]
This must be re-proposed against the `mitaka` branch if still required.

**Pull request closed without merging**